### PR TITLE
Release notes for 2.0.18

### PR DESCRIPTION
**Pull request by derekbekoe**

### CODE REVIEW

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[latest/docs-ref-conceptual/release-notes-azure-cli.md](https://github.com/derekbekoe/azure-docs-cli-python/blob/cli-release-2.0.18/latest/docs-ref-conceptual/release-notes-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/azure-cli-latest/release-notes-azure-cli?view=azure-cli-latest&branch=pr-en-us-498) (azure-cli-latest) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C9%5C23%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201709230005417047-498%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ikganD32C%2BhygBbEczfyS2vyB46cG5P%2FiRDdwdLMELc%3D&st=2017-09-23T00%3A03%3A04Z&se=2017-09-24T00%3A13%3A04Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: latest/docs-ref-conceptual/release-notes-azure-cli.md [hunk 1]
```diff
@@ -16,6 +16,66 @@ ms.assetid: ce0428f7-0a59-4e72-9237-d907b171af51
...
+## September 22, 2017
+
+Version 2.0.18
+
+### Resource
+
+* Added support for showing built-in policy definitions
+* Added support mode parameter for creating policy definitions
+* Added support for UI definitions and templates to `managedapp definition create`
+* BREAKING CHANGE: managedapp: Update to latest ARM package, which includes changing resource type from appliances to applications and applianceDefinitions to applicationDefinitions
```
Comment (sptramer) [inline latest/docs-ref-conceptual/release-notes-azure-cli.md]: @derekbekoe - I'm not sure if this is an actual user-facing change or not. Couldn't create a test managed app to check. Can you remove or improve this line?

File: latest/docs-ref-conceptual/release-notes-azure-cli.md [hunk 2]
```diff
@@ -16,6 +16,66 @@ ms.assetid: ce0428f7-0a59-4e72-9237-d907b171af51
...
+* Added `--ddos-protection` and `--vm-protection` arguments to `vnet [create|update]`
+* Added `network [vnet-gateway|vpn-client|show-url]` commands
+
+### Storage
+
+* Fixed issue where `storage account network-rule` commands may fail after updating the SDK
+
+### Eventgrid
+
+* Updated eventgrid SDK to 0.2.0
```
Comment (sptramer) [inline latest/docs-ref-conceptual/release-notes-azure-cli.md]: Replace with changes that affect CLI users

File: latest/docs-ref-conceptual/release-notes-azure-cli.md [hunk 3]
```diff
@@ -16,6 +16,66 @@ ms.assetid: ce0428f7-0a59-4e72-9237-d907b171af51
...
+
+### SQL
+
+* Changed `sql server list` argument `--resource-group` to be optional. If not specified, all sql servers in the subscription will be returned
+* Added `--no-wait` param to `db [create|copy|restore|update|replica create|create|update]` and `dw [create|update]`
+
+### Keyvault
+
+* Updated azure-keyvault SDK to 0.3.6
```
Comment (sptramer) [inline latest/docs-ref-conceptual/release-notes-azure-cli.md]: Replace with changes that affect CLI users

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[latest/docs-ref-conceptual/release-notes-azure-cli.md](https://github.com/derekbekoe/azure-docs-cli-python/blob/cli-release-2.0.18/latest/docs-ref-conceptual/release-notes-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/azure-cli-latest/release-notes-azure-cli?view=azure-cli-latest&branch=pr-en-us-498) (azure-cli-latest) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C9%5C25%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201709251756194147-498%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=5R6i6rDJwv1ON98%2FAvxZg8QVQQ1dfY3o5qBUVouUDgA%3D&st=2017-09-25T17%3A56%3A59Z&se=2017-09-26T18%3A06%3A59Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: latest/docs-ref-conceptual/release-notes-azure-cli.md (hunk 3 above)
Comment (derekbekoe) [reply latest/docs-ref-conceptual/release-notes-azure-cli.md]: done

File: latest/docs-ref-conceptual/release-notes-azure-cli.md (hunk 1 above)
Comment (derekbekoe) [reply latest/docs-ref-conceptual/release-notes-azure-cli.md]: done

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[latest/docs-ref-conceptual/release-notes-azure-cli.md](https://github.com/derekbekoe/azure-docs-cli-python/blob/cli-release-2.0.18/latest/docs-ref-conceptual/release-notes-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/azure-cli-latest/release-notes-azure-cli?view=azure-cli-latest&branch=pr-en-us-498) (azure-cli-latest) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C9%5C25%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201709251949178330-498%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=YC8CaiyEdjWYsDxYV36Nx7fpmPQ%2FXGKHZYiudGo47j0%3D&st=2017-09-25T19%3A47%3A17Z&se=2017-09-26T19%3A57%3A17Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[latest/docs-ref-conceptual/release-notes-azure-cli.md](https://github.com/derekbekoe/azure-docs-cli-python/blob/cli-release-2.0.18/latest/docs-ref-conceptual/release-notes-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/azure-cli-latest/release-notes-azure-cli?view=azure-cli-latest&branch=pr-en-us-498) (azure-cli-latest) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C9%5C25%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201709252028115421-498%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=KBhj%2FH7Ohf7xztJJmxWFeMeAxviwu8ua7tkGm7fRxic%3D&st=2017-09-25T20%3A26%3A32Z&se=2017-09-26T20%3A36%3A32Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**